west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8a249246bc2add52db9d1b9ee5db111e0ccda4c3
+      revision: c0db766c8274040babc30ce3e79b4e881adae097
```

Comment:
SHA was updated in https://github.com/nrfconnect/sdk-nrf/pull/9073 to a more recent revision, please drop this change.

Reply:
Done